WebRoot/h/changepass

```diff
@@ -81,53 +77,53 @@
-                                <td class='zLoginLabelContainer'><label for="oldPassword"><fmt:message key="oldPassword"/>:</label></td>
                                 <td class='zLoginFieldContainer'>
-                                    <input id="oldPassword" autocomplete="off" class='zLoginField' name='password' type='password' value="" maxlength="${domainInfo.webClientMaxInputBufferLength}"/>
+                                    <input id="oldPassword" placeholder="<fmt:message key='oldPassword'/>" autocomplete="off" class='zLoginField' name='password' type='password' value="" maxlength="${domainInfo.webClientMaxInputBufferLength}"/>
                                 </td>
                             </tr>
                             <tr>
-                                <td class='zLoginLabelContainer'><label for="newPassword"><fmt:message key="newPassword"/>:</label></td>
                                 <td class='zLoginFieldContainer'>
-                                    <input id="newPassword" autocomplete="off" class='zLoginField' name='loginNewPassword' type='password' value="" maxlength="${domainInfo.webClientMaxInputBufferLength}"/>
+                                    <input id="newPassword" placeholder="<fmt:message key='newPassword'/>" autocomplete="off" class='zLoginField' name='loginNewPassword' type='password' value="" maxlength="${domainInfo.webClientMaxInputBufferLength}"/>
                                 </td>
                             </tr>
                             <tr>
-                                <td class='zLoginLabelContainer'><label for="confirm"><fmt:message key="confirm"/>:</label></td>
-                                <td class='zLoginFieldContainer'>
-                                    <input id="confirm" autocomplete="off" class='zLoginField' name='loginConfirmNewPassword' type='password' value="" maxlength="${domainInfo.webClientMaxInputBufferLength}"/>
+                                <td class='zLoginFieldContainer ZChangePassConfirm'>
+                                    <input id="confirm" placeholder="<fmt:message key='confirm'/>" autocomplete="off" class='zLoginField' name='loginConfirmNewPassword' type='password' value="" maxlength="${domainInfo.webClientMaxInputBufferLength}"/>
                                 </td>
                             </tr>
+
+                            <c:if test="${not empty errorMessage}">
+                                <tr>
+                                    <td class='errorText'>
+                                        <c:out value="${errorMessage}"/>
+                                    </td>
+                                </tr>
+                            </c:if>
+
                             <tr>
-                                <td></td>
-                                <td><input type=submit class='zLoginButton'
+                                <td class='ZChangePassButton'><input type=submit class='zLoginButton'
```

Comment:
In some my previous PR, I was suggested not to use last-child, so avoided doing that.
The reason being if the skin develop changes templates, it may affect the layout.
So, not using that.
@silentsakky , what would you say, here.